### ISSUE TITLE
[Suggestion]: Wither Storm and Wither Storm Segments break small blocks when falling from getting hit with the Formidibomb

### ISSUE THREAD

**Issue by TaintedBlock** [nonamecrackers2/crackers-wither-storm-mod#1690]
### Description of feature request

I notice when you bomb the Wither Storm, it will fall and land on anything that it touches. There are some case where there are small blocks in the way of the Wither Storm, and it tends to land on these and it always looks like it just landed in the air. The alternative to this, is to have the Wither break small blocks in the area when falling on them so it can land on a decently bigger area. This goes for the segments too of course.

**Comment by Pandaboi25x** [comment]
FUCK I WANTED TO SUGGEST IT wait nvm i didnt

**Comment by CubeGamer647** [comment]
Yes, it's always annoying when there's a single leaf in the corner of the hotbox and it lands on that somehow

**Comment by GearsawStudios** [comment]
Perhaps it should keep falling and destroy grass, dirt and sand so it looks like it crushed the ground.
